demo/nodejs/file.js

```diff
@@ -55,10 +63,14 @@ function fileDemo() {
   }
 
   try {
-    console.log(engineInstance.processFile(audioPath));
+    const res = engineInstance.processFile(audioPath);
+    console.log(res.transcript);
+    if (verbose) {
+      console.table(res.words);
```

Comment:
I don't know how much control you have over it, but when I tried the demo, I got a lot of decimal places in the outputs for each float value e.g.) `0.5759999752044678`. It would be better if we could round them or print them at lower precision as it's a bit hard to decode.

Reply:
unfortunately I have no control on the style with `console.table`. If it's too bad, I can implement my own log system.

Reply:
Fixed :) 
![image](https://user-images.githubusercontent.com/42750891/183154119-58c1ae9a-ff9a-484a-a1a4-080eccedc266.png)